sunpy/database/tests/test_tables.py

```diff
@@ -99,10 +99,10 @@ def test_entry_from_qr_block(query_result):
     entry = DatabaseEntry._from_query_result_block(query_result[0])
     expected_entry = DatabaseEntry(
         source='SOHO', provider='SDAC', physobs='intensity',
-        fileid='/archive/soho/private/data/processed/eit/lz/2001/01/efz20010101.010014',
-        observation_time_start=datetime(2001, 1, 1, 1, 0, 14),
-        observation_time_end=datetime(2001, 1, 1, 1, 0, 21),
-        instrument='EIT', size=2059.0, wavemin=17.1, wavemax=17.1)
+        fileid='/archive/soho/private/data/processed/eit/lz/2001/01/efz20010101.000042', #010014
+        observation_time_start=datetime(2001, 1, 1, 0, 0, 42), #(2001, 1, 1, 1, 0, 14)
+        observation_time_end=datetime(2001, 1, 1, 0, 0, 54), #datetime(2001, 1, 1, 1, 0, 21)
+        instrument='EIT', size=2059.0, wavemin=19.5, wavemax=19.5) #wavemin=17.1, wavemax=17.1
```

Comment:
I'm not too convinced on writing coments to keep an eye on what was in the past... Mostly in the current way. If someone looks at this file without knowing that these comments are an old result given by the server... then it's very confusing. Could you write a line below the changes a bit more explanatory? Or not comment at all and add these changes in an issue where we can always point to and reopen when necessary, so it's easier to track when the differences were spotted and inform the data providers (eventually finding out where the bug is happening).